pkgs/top-level/all-packages.nix

```diff
@@ -1886,7 +1886,7 @@ with pkgs;
 
   kitty = callPackage ../applications/terminal-emulators/kitty {
```

Comment:
Consider `darwin.apple_sdk_11_0.callPackage` instead of `callPackage` here? 
Per https://nixos.org/manual/nixpkgs/unstable/#sec-darwin behaves the same as pure `callPackage` on Linux, but forces new SDK on darwin.

Reply:
Done.